### PR TITLE
adds confetti and share popup

### PR DESCRIPTION
**Pull request by nitinshr135**
Adds 
- Confetti on successful payment
- Share button to share the deployment

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,12 +80,15 @@
     "react-scripts": "3.4.3",
     "react-spinners": "^0.9.0",
     "react-tooltip": "^4.2.21",
+    "react-use": "^17.2.4",
```
Comment (rekpero) [inline package.json:83]: Is this package anywhere used? If not remove it and run `npm install` or `yarn install`

File: package.json [hunk 2]
```diff
@@ -57,18 +57,21 @@
...
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@react-hook/window-size": "^3.0.7",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "bnc-notify": "^1.6.1",
     "bnc-onboard": "^1.25.0",
     "broadcast-channel": "^3.2.0",
+    "canvas-confetti": "^1.4.0",
```
Comment (rekpero) [inline package.json:67]: Is this also anywhere used? Otherwise remove it and run `npm install` or `yarn install`

File: package.json [hunk 3]
```diff
@@ -77,12 +80,15 @@
...
     "react-tooltip": "^4.2.21",
+    "react-use": "^17.2.4",
+    "reactjs-popup": "^2.0.5",
     "rxjs": "^6.6.2",
     "socket.io-client": "^4.1.1",
     "source-map-explorer": "^2.5.0",
     "typescript": "~3.7.2"
   },
   "devDependencies": {
+    "@types/canvas-confetti": "^1.4.2",
```
Comment (rekpero) [inline package.json:91]: same with this as well

Comment (rekpero) [reply package.json:91]: @nitinshr135 make sure you clean up any packages before commit if they are not getting used anywhere

File: src/components/Sites/routes/Deployment/Deployment.tsx [hunk 4]
```diff
@@ -579,6 +624,90 @@ const Deployment = () => {
...
+                    >
+                      Share
+                    </button>
+                  }
+                  position="center center"
+                  open={confettiStart}
+                  className="popup-container"
+                  modal
+                >
+                  <div className="modal-container">
```
Comment (rekpero) [inline src/components/Sites/routes/Deployment/Deployment.tsx:644]: Please create a different component for this modal and replace this code with a component and pass any data thru props.

**Review by rekpero — CHANGES_REQUESTED** [review]
LGTM! @nitinshr135 please resolve the comments

File: src/components/Sites/routes/Deployment/Deployment.tsx (hunk 4 above)
Comment (nitinshr135) [reply src/components/Sites/routes/Deployment/Deployment.tsx]: added different component for modal, cleaned code

**Review by rekpero — APPROVED** [review]
LGTM!!

**Pull request merged**